.github/workflows/ci.yml

```diff
@@ -32,7 +32,7 @@ concurrency:
 jobs:
   build:
 
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Should be `ubuntu-24.02`?

Reply:
The latest stable for this CI should be fine.